java_v1/App.java

```diff
@@ -0,0 +1,52 @@
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashSet;
+
+public class App {
+	
+	public static void main(String[] args) {
+
+		URL url = null;
+		HttpURLConnection conn = null;
+		HashSet<String> nodes = new HashSet<String>();
+		
+		String path = "https://www.raspberrypi.org/blog/the-little-computer-that-could/";
```

Comment:
Good idea. I changed it to read from an env variable and re-submitted another PR.